Authorization/Authorization.xcodeproj/project.pbxproj

```diff
@@ -1430,7 +1454,7 @@
 					"@loader_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = org.openedx.Authorization;
+				PRODUCT_BUNDLE_IDENTIFIER = org.openedx.Authorization.nelc;
```

Comment:
Please return old PRODUCT_BUNDLE_IDENTIFIER and DEVELOPMENT_TEAM

Reply:
✅